src/main/java/gregtech/api/recipes/RecipeBuilder.java

```diff
@@ -680,8 +680,7 @@ protected EnumValidationResult finalizeAndValidate() {
 
     public ValidationResult<Recipe> build() {
         return ValidationResult.newResult(finalizeAndValidate(), new Recipe(inputs, outputs, chancedOutputs,
-                fluidInputs, fluidOutputs, duration, EUt, hidden, isCTRecipe,
-                        recipePropertyStorage == null ? EmptyRecipePropertyStorage.INSTANCE : recipePropertyStorage));
+                fluidInputs, fluidOutputs, duration, EUt, hidden, isCTRecipe, recipePropertyStorage));
```

Comment:
Does this change also need to be made in `GasCollectorRecipeBuilder#getDimensionIDs`?

Reply:
There's an independent null check over there, so it'll be fine